### ISSUE TITLE
Problems with Vuex Timetravel because of MomentJs

### ISSUE THREAD

**Issue by michastubi** [MatheusrdSantos/vue-quick-chat#6]
I had some problems in my App when i tried to use Vuex Timetravel. The Problems are the Moment objects in the Vuex Store. 
I have solved it by saving the ISO 8601 String of the timestamp in the Vuex Store instead of the Moment object. 
Then I also added a getter method which converts the String to a moment Object. 
The changes  for this are quite simple:
Here are the new mutatiuons:

```javascript
newMessage: (state, message) => {
            message.timestamp = message.timestamp.toISOString();
            state.messages = [...state.messages, message];
        },
setMessages: (state, messages) => {
            messages.map(message => {
                message.timestamp = moment(message.timestamp).toISOString();
            })
            state.messages = messages;
        }
```
and here is the new getter:
```javascript
  messages: (state) => {
            let messages = [];
            state.messages.forEach(message => {
                let newMessage = { ...message };
                newMessage.timestamp = moment(newMessage.timestamp);
                messages.push(newMessage);
            })
            return messages;
        }
```
There are some other ways to solve this problem. Here is a very good article:
https://medium.com/dailyjs/dates-in-vuex-primitives-or-complex-objects-6e6b29ebb880
At the bottom there is even an paragraph about momentjs.

Greetings

**Comment by MatheusrdSantos** [comment]
Hi, @michastubi! Thanks, for report this issue!
Great article you shared, really usefull.
I will follow your suggestion and solve this problem in the next few days.
However, if your prefer, feels free to fork this project and implement the solution.
Greetings :)

**Comment by MatheusrdSantos** [comment]
Hi, @michastubi. 
The problem was solved in this pull request #7. So I'll close this issue.
Thanks again!
